src/bootlint.js

```diff
@@ -1188,6 +1188,9 @@ var LocationIndex = _location.LocationIndex;
                         seenLint = true;
                     }
 
+                    // Append the wiki URL to the lint message.
```

Comment:
Sounds good, @cvrebert! Pushed an update. Let me know if you have concerns. I didn't rebase since Github has the fancy new squashing feature :)